aten/src/ATen/native/native_functions.yaml

```diff
@@ -4197,33 +4197,45 @@
     CPU: legacy::cpu::_th_ior_
     CUDA: legacy::cuda::_th_ior_
 
+- func: bitwise_xor.Tensor_out(Tensor self, Tensor other, *, Tensor(a!) out) -> Tensor(a!)
+  variants: function
+  dispatch:
+    CPU: bitwise_xor_out
```

Comment:
Hmmm if they're dispatching to the same function for different devices, then the dispatch is not really necessary?

Reply:
I think this will give an error message on device not supported, such as quantized. Am I right on this?

Reply:
Oh! Hmmm I think creating device dispatch is an overkill to get error message, @VitalyFedyunin do you know how do we usually handle this ?